po/ja.po

```diff
@@ -0,0 +1,561 @@
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:159
+msgid "Images support"
+msgstr "画像対応"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:165
+msgid "Trim items"
+msgstr "項目をトリムする"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:171
+msgid "Detect growing lines"
+msgstr "成長する線を検出する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:178
+msgid "Save history"
+msgstr "履歴を保存する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:201
+msgid "Max element size when displaying"
+msgstr "表示時の最大要素大きさ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:208
+msgid "Max displayed history size"
+msgstr "最大表示履歴大きさ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:215
+msgid "Max history size"
+msgstr "最大履歴大きさ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:222
+msgid "Max memory usage (MB)"
+msgstr "最大メモリ使用 (MB)"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:229
+msgid "Max text item length"
+msgstr "最大テキスト項目の長さ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:236
+msgid "Min text item length"
+msgstr "最小テキスト項目の長さ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:262
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:72
+msgid "Delete the active item from history"
+msgstr "履歴からアクティブな項目を削除する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:269
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:73
+msgid "Launch the graphical tool"
+msgstr "GUI ツールを起動する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:276
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:74
+msgid "Mark the active item as being a password"
+msgstr "アクティブな項目をパスワードとしてマークする"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:283
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:75
+msgid "Display the history"
+msgstr "履歴を表示する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:290
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:76
+msgid "Sync the clipboard to the primary selection"
+msgstr "クリップボードを一次選択に同期する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:297
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:77
+msgid "Sync the primary selection to the clipboard"
+msgstr "一次選択をクリップボードに同期する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:304
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:78
+msgid "Upload the active item to a pastebin service"
+msgstr "ペーストビン・サービスにアクティブな項目をアップデートする"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:319
+#: src/libgpaste/ui/gpaste-ui-window.c:311 src/client/gpaste-client.c:789
+msgid "Couldn't connect to GPaste daemon"
+msgstr "GPaste デーモンに接続できませんでした"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:334
+msgid "General behaviour"
+msgstr "一般行動"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:335
+msgid "History settings"
+msgstr "履歴設定"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:336
+msgid "Keyboard shortcuts"
+msgstr "キーボード・ショートカット"
```

Comment:
```suggestion
msgstr "キーボードショートカット"
```

Yours IS correct, but IIRC this translation is used in other GNOME components.

Reply:
I see.